geometry_msgs/Pose Pose
---
bool res